tsconfig.json

```diff
@@ -30,7 +30,8 @@
       "@app/*": ["src/app/*"],
       "crypto": ["node_modules/crypto-browserify"],
       "stream": ["node_modules/stream-browserify"],
-      "zlib": ["node_modules/browserify-zlib"]
+      "zlib": ["node_modules/browserify-zlib"],
+      "@asam": ["node_modules/@dvsa/capacitor-plugin-asam"]
```

Comment:
Do we think this is necessary / has benefits? Other ```@dvsa/*``` packages are not aliased in such way

Reply:
this is the way Capacitor recomends injecting plugin mocks https://capacitorjs.com/docs/guides/mocking-plugins#jasmine

Reply:
Maybe i have misinterpreted then as to me there should be no need in this file, but in the tsconfig.spec.json you would have

```"@dvsa/capacitor-plugin-asam": "@mocks/capacitor/asam.ts"```

Reply:
updated as discussed